lib/kase/switcher.rb

```diff
@@ -29,7 +29,7 @@ def on(*pattern)
     end
 
     def validate!
-      raise NoMatchError unless matched?
+      raise NoMatchError.new(@values) unless matched?
```

Comment:
Please use the attr_reader `values` instead of `@values` here. If the reader is changed to be a method, this should probably follow that change.